Components/Modal/Modal.css

```diff
@@ -0,0 +1,89 @@
+.dialogs{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.dialog-container1{
+    border: 0.1rem solid var(--black-font-color);
+    border-radius: 2rem;
+    width:  40rem;
+    height: 20rem;
+    padding: 0 0 2rem 0;
+    box-shadow: 0.5rem 1rem var(--gray-border);
+
+
+   
+}
+
+.dialog-heading{
+
+    font-size: larger;
+    color: var(--black-font-color);
+    font-weight: bold;
+   
+}
+
+.dialog-info{
+  width: 30rem;
+} 
+
+
+
+
+.dialog-btn{
+  
+    border-style: none;
+    color: var(--dialog-btn-color);
+    font-size: large;
+    font-weight: bold;
```

Comment:
Suggestion , You can give here some value like `500` , `600` ,etc

Reply:
okay will try using it